cmd/txncleaner/main.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	mgo "gopkg.in/mgo.v2"
+
+	"github.com/juju/txn"
+)
+
+var dbName = flag.String("db", "", "mongo database name (required)")
+var txnsName = flag.String("txns", "txns", "mgo txns collection name")
+var url = flag.String("url", "localhost:27017", "mongo URL")
+
+func main() {
+	flag.Parse()
+
+	if *dbName == "" {
+		flag.PrintDefaults()
+		os.Exit(1)
+	}
+
+	session, err := mgo.Dial(*url)
+	if err != nil {
+		log.Fatalf("failed to connect to mongo: %v", err)
+	}
+
+	db := session.DB(*dbName)
+	txnsC := db.C(*txnsName)
+
+	txnCount, err := txnsC.Count()
```

Comment:
you don't need to do this, CleanAndPrune will do it if necessary (hence the param is "*Hint"). It was only passed in because 'maybePrune' already needed the information.